Netkan/Processors/QueueHandler.cs

```diff
@@ -211,6 +214,14 @@ private SendMessageBatchRequestEntry inflationMessage(Metadata ckan, Metadata ne
                         DataType    = "String",
                         StringValue = DateTime.UtcNow.ToString("s", CultureInfo.InvariantCulture)
                     }
+                },
+                {
+                    "Game",
```

Comment:
In terms of the outbound message attribute, I think `GameId` would be appropriate and fits with the SQS structure. It is what informed the design for the multi-game support in KSP-CKAN/NetKAN-Infra#283

https://github.com/KSP-CKAN/NetKAN-Infra/blob/master/netkan/netkan/webhooks/spacedock_add.py#L45
https://github.com/KSP-CKAN/NetKAN-Infra/blob/master/netkan/netkan/webhooks/github_mirror.py#L58
https://github.com/KSP-CKAN/NetKAN-Infra/blob/master/netkan/netkan/metadata.py#L98

```suggestion
                    "GameId",
```